Program.cs

```diff
@@ -1,7 +1,20 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Input: ");
+         string decimal_number = Console.ReadLine ();
+         if (int.TryParse (decimal_number, out int dec) && dec!=0) {
+            string hexa = dec.ToString ("X");
+            Console.WriteLine ($"HEX: {hexa}");
+            string bin = "";
+            while (dec > 0) {
+               int rem = dec % 2;
+               bin = rem + bin;

```

Comment:
Since "rem" is not used only in one place, it could've been used straight away in the "bin" assignment line.

Reply:
Remainder used as expression.